### DISCUSSION TITLE
TextLayer labels hidden inside 3d-geometry

### DISCUSSION THREAD

**Discussion by yaras-phoenix** [visgl/deck.gl#8779] [General]
Problem in the topic can be described with this image:
![image](https://github.com/visgl/deck.gl/assets/37367584/53f827a6-d257-4432-9918-2d91aa5d7ce7).

What I'd like to achieve is 'labels always visible'.

Using demo-layers I made [this codepen](https://codepen.io/yskiugsp-the-looper/pen/XWQqxdG?editors=001).

I've  found similar issue [here](https://github.com/visgl/deck.gl/issues/7428) and posted a comment. [ibgreen](https://github.com/ibgreen) responded but I already have depth testing set to false in my codepen

**Comment by ibgreen** [comment]
Try rendering your texts after your other layers.
```
  layers: [layer, layer_text]
```

**Reply by yaras-phoenix** [reply]
![image](https://github.com/visgl/deck.gl/assets/37367584/769b6ea1-e82e-407a-a24a-be4cc4a92ce9)
Worked like charm, thank you!
